### PR TITLE
homebrew: Bump beta version

### PR DESCRIPTION
**Pull request by mberhault**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5715)

### CODE REVIEW

**Comment by petermattis** [comment]
LGTM

File: build/cockroach.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Cockroach < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
   url "https://github.com/cockroachdb/cockroach.git",
-      :tag => "beta-20160329",
-      :revision => "7d356a93316165f36c3f772f80caabaa2dd29749"
+      :tag => "beta-20160330",
+      :revision => "ebf4980be08fd0fe70cb3863a6053543f816cedc"
```
Comment (tamird) [inline build/cockroach.rb]: go ahead and remove this - it's superfluous. (yeah, most formulae have both, but i can't see why - i read the code, i believe it's unused).

Comment (mberhault) [inline build/cockroach.rb]: the revision? I'm happy to, but it may be best done by someone also running the thing. I don't even know what this file does.

Comment (petermattis) [inline build/cockroach.rb]: I think I tried to remove it at one point and something didn't work. I don't recall what. Feel free to give it another go.

Comment (tamird) [inline build/cockroach.rb]: Ah, it's intentional - they verify that the tag hasn't been changed.

**Pull request merged**